docs/user-docs/openqa.md

```diff
@@ -0,0 +1,244 @@
+- screenshots and a video recording of the screen for the duration of a test
+- the results and the reasons behind fails
+
+Anyone can login to the website via GitHub, but only the authorized
+accounts are able to:
+- run the tests
+- create new tests
+- modify any settings
+- add new devices
+
+## Navigation
+
+The main page of the OpenQA website shows a summary of the most recent test
+runs.
+
+The tests are organized into _Job Groups_, like "Qubes".
+The test groups contain _Products_, like "Build4.2.3".
+Each _Product_ consists of multiple _Test Suites_. On the main page
+every distinct _Product_ has a progress bar showing how many _Test Suites_
+have passed, are running, are finished and were skipped.
+
+|![Image of the main page](../img/openqa/main-page.png)
+|:-:|
+|Image of the main page, show job groups, products and test suite progress|
+
+From this page there is a couple of options for further navigation to access
+more precise data.
+
+### Job Group
+
+_Job Groups_ simply a way to organize all of the maintained _Products_ into
+thematic categories.
+
+Clicking on the name of a _Job Group_ shows its details. Most importantly all
+of the historical _Products_ that were published as a part of the _Job Group_.
+In addition to that, authorized users can leave comments about the _Job Group_
+there.
+
+|![Image of the Job Group view](../img/openqa/job-group-view.png)
+|:-:|
+|Image of the Job Group view|
+
+### Product
+
+A _Product_ represents one run of a set of test suites with given parameters.
+The name of the products consists of a couple parts
+It consists of:
+- The name of the tested OS distribution - to run tests on every maintained
+  distribution
+- The version - to allow running the tests on every new release
+- The flavor - to allow testing every flavor of the OS separately
+- The hardware architecture - to run on every supported architecture
+
+Testing TrenchBoot requires to cover all of the supported hardware platforms,
+not only run a single test per hardware architecture. For this reason the
+_Flavor_ field of a _Product_ is used a bit unconventionally. At this moment
+TrenchBoot does not consist of multiple _Flavors_ and the field is used to
+determine:
+- on which hardware platform the tests were performed
+- whether it was installed on a Legacy or an UEFI BIOS
+
+|![Product name made of four sections](../img/openqa/medium-string.png)|
+|:-:|
+|Product name made of four sections|
+
+Clicking on a _Product_ from the main page, or the _Job Group_ details page
+shows the summary of the _Product_ and a list of the _Test Suites_ associated
+with it.
+
+|![Image of a product view with a list of associated tests](../img/openqa/product-view.png)|
+|:-:|
+|Image of a product view with list of tests|
+
+### Test Suite
+
+_Test Suites_ are the essence of the OpenQA web panel. Next to every test suite
+you can see a small colorful circle informing about the status of the
+_Test Suite_. The most common statuses are:
+- Gray - Skipped
```

Comment:
User cancelled tests are also of gray color (can be disambiguated via a tooltip text).